src/screens/Home/Home.js

```diff
@@ -83,9 +68,10 @@ export default function Home () {
           <Box height='2em' width='12em'>
             <Image alt="Powered by Zooniverse" fit='contain' src={Zooniverse} />
           </Box>
-          <StyledHeader >
-            Transcription viewer/editor
-          </StyledHeader>
+          <Box width='50%'>
+            <Image fit='contain' src={Alice} />
```

Comment:
This requires an `alt` tag and we're all good.

Reply:
Oops! Good catch.